PostHog/PostHogConfig.swift

```diff
@@ -26,6 +26,7 @@ import Foundation
     @objc public var captureScreenViews: Bool = true
     @objc public var debug: Bool = false
     @objc public var optOut: Bool = false
+    @objc public var getDeviceId: ((UUID) -> UUID) = { uuid in uuid }
```

Comment:
@jimexist this function is actually used for the `$device_id` property and not the `$anon_distinct_id` property (in JS).
if the intention is that the anonymous ID is configurable, it will make more sense to call it `getAnonymousId` instead

Reply:
fixed and pushed